### DISCUSSION TITLE
/html/images/spacer.gif

### DISCUSSION THREAD

**Discussion by theking2** [LMS-Community/slimserver#1093] [General]
looks like this on my Lyrion 9.0.0 server. That seems wrong. Do I post this as an issue?

![image](https://github.com/LMS-Community/slimserver/assets/1612152/ac8405cb-e783-497c-969b-50a61bfbb09c)

**Comment by michaelherger** [comment]
Where do you see this?

**Comment by theking2** [comment]
here 

![image](https://github.com/LMS-Community/slimserver/assets/1612152/172c149e-ab5d-4b2c-874c-2524c897e63b)

**Comment by michaelherger** [comment]
Are you using the "Dark Default" skin? That's a 3rd party plugin which might need some update.

https://github.com/AF-1/lms-darkdefaultskin/tree/main

**Reply by theking2** [reply]
ah, ok. is there a build in dark theme? btw I use the docker image so the theme in the docker image (9.0.0) perhaps is not up to date?

**Reply by terual** [reply]
If you use a third party skin I think it is saved in the config folder and thus is not part of the docker image.

**Reply by theking2** [reply]
Didn't think of that🤦‍♂️, sorry you are right.
